be2-scala/src/test/scala/ch/epfl/pop/decentralized/MonitorSuite.scala

```diff
@@ -185,23 +181,21 @@ class MonitorSuite extends TestKit(ActorSystem("MonitorSuiteActorSystem")) with
     mockConnectionMediator.send(monitorRef, ConnectionMediator.Ping())
 
     // Expect first read of the server peers list
-    mockConnectionMediator.expectMsgType[ConnectionMediator.ConnectTo](timeout)
+    mockConnectionMediator.expectMsgType[ConnectionMediator.ConnectTo](20.seconds)
```

Comment:
Might want to create a new constant value or use an existing one, so that we can have consistency on the desired timeout for this case on future tests.